frb_example/pure_dart/dart/lib/bridge_generated.web.dart

```diff
@@ -0,0 +1,1971 @@
+// AUTO GENERATED FILE, DO NOT EDIT.
+// Generated by `flutter_rust_bridge`.
+
+// ignore_for_file: non_constant_identifier_names, unused_element, duplicate_ignore, directives_ordering, curly_braces_in_flow_control_structures, unnecessary_lambdas, slash_for_doc_comments, prefer_const_literals_to_create_immutables, implicit_dynamic_list_literal, duplicate_import, unused_import, prefer_single_quotes, prefer_const_constructors, use_super_parameters, always_use_package_imports, annotate_overrides
+
+import "bridge_definitions.dart";
+import 'dart:convert';
+import 'dart:typed_data';
+import 'package:flutter_rust_bridge/flutter_rust_bridge.dart';
+
+class FlutterRustBridgeExampleSingleBlockTestImpl
+    extends FlutterRustBridgeBase<FlutterRustBridgeExampleSingleBlockTestWire>
+    implements FlutterRustBridgeExampleSingleBlockTest {
+  factory FlutterRustBridgeExampleSingleBlockTestImpl(WasmModule dylib) =>
+      FlutterRustBridgeExampleSingleBlockTestImpl.raw(FlutterRustBridgeExampleSingleBlockTestWire(dylib));
+
+  FlutterRustBridgeExampleSingleBlockTestImpl.raw(FlutterRustBridgeExampleSingleBlockTestWire inner) : super(inner);
+
+  Future<int> simpleAdder({required int a, required int b, dynamic hint}) => executeNormal(FlutterRustBridgeTask(
+        callFfi: (port_) => inner.wire_simple_adder(port_, _api2wire_i32(a), _api2wire_i32(b)),
+        parseSuccessData: _wire2api_i32,
+        constMeta: kSimpleAdderConstMeta,
+        argValues: [a, b],
+        hint: hint,
+      ));
```

Comment:
From a very quick look, seems that all such functions are the same as the non-web counterpart?

If so, maybe we do not need to generate such code twice, but only need to generate things like `_api2wire_i32` twice - since we know duplication may not be best

As for `_api2wire_i32` is private - what about using `part` and `part of` keyword

Reply:
I wouldn't mind factoring out the similar api2wire code, at the expense of some added complexity to our existing code.

> As for _api2wire_i32 is private - what about using part and part of keyword
That's a great idea, I might try it out actually.

Reply:
Sadly `part of` cannot be declared conditionally based on targets, so fallback to normal imports.